frontend-ssr/styles/utils.scss

```diff
@@ -32,6 +32,9 @@
 .no-underline {
   text-decoration: none;
 }
+.underline-on-hover:hover {
```

Comment:
Just checked `tailwind.css` and I think we should name it `hover:underline`. It's the way they name it and it might be easier for people familiar with that library.

PS: mentioned this in CONTRIBUTING.md as well

Reply:
Done